### PR TITLE
Wrapper for classification #38

### PR DESCRIPTION
**Pull request by apoman38**
**What this PR does:**

Added a new file _wrapper.py._ This file receives an mp4 file as input and returns in which class it belongs to.

In the _train.py_ file removed the one-hot encoding of the labeled data and added the saving of the trained model.

In the _analyze_visual.py_ added saving feature extraction in .npy format (not only in .csv).

**How to QA:**

Run the script `python3 wrapper.py -f dataset/trump.mp4 -m SVM` where -f option is the name of the file and -m the saved model of the training algorithm. The script must return the class that the file belongs.

### CODE REVIEW

**Comment by tyiannak** [comment]
So I guess I need to retrain the model as well right?

**Comment by apoman38** [comment]
Yes right, because in the previous version train.py didn't save models.

File: analyze_visual/train.py [hunk 1]
```diff
@@ -104,10 +105,6 @@ def data_preparation(x):
     # fit and transform the data
     x_all = scaler.fit_transform(x_all)
```
Comment (tyiannak) [inline analyze_visual/train.py:106]: This is the wrong point to use the scaler and it causes a bug in wrapper. 
The scaler must be fitted on the training data and then used during evaluation. Also the scaler must be embedded in the model and saved in the file (and obviously loaded in the wrapper). 
Please @apoman38 fix this bug.

**Review by tyiannak — CHANGES_REQUESTED** [review]

File: analyze_visual/wrapper.py [hunk 2]
```diff
@@ -14,53 +14,72 @@
...
-        features_stats = process_video(shot, 2, True, True, True)
-        features = features_stats[0]
-        features = features.reshape(1,-1)
-
-    model_predict(features,model)
+    videos_path = args.videos_path
+    algorithm = args.model
+
+    # Convert list of lists to a single list
+    videos_path = [item for sublist in videos_path for item in sublist]
```
Comment (tyiannak) [inline analyze_visual/wrapper.py:73]: Please check if this is a file or folder. If folder do what u'r doing now. If file, just run the wrapper on a single video

**Review by tyiannak — CHANGES_REQUESTED** [review]

**Pull request merged**